webapp/index.html

```diff
@@ -10,11 +10,11 @@
 		data-sap-ui-libs="sap.m"
 		data-sap-ui-xx-waitForTheme="true"
```

Comment:
...-xx-wait-for-theme (not on the linter's list, I guess)

Reply:
Done